CONTRIBUTING.md

```diff
@@ -112,58 +129,69 @@ you can contact Jessie Young. Otherwise, you can create a Slack bot
+We are using [OmniAuth GitHub Team
+Auth](https://github.com/jonmagic/omniauth-github-team-member) for
+authentication.
 
-If you are internal to 18F, contact Brian Hedberg to be added to the developer
-list.  If you are on the list, `dolores-local` will be one of your [Authorized
-Applications](https://alpha.my.usa.gov/authorizations) on MyUSA.
+1. Create a [GitHub OAuth application](https://developer.github.com/v3/oauth/)
+for development with the following:
 
-If `dolores-local` is on your MyUSA list for Authorized Applications and you
-are still unable to authenticate, check with Brian to make sure that the `MYUSA_KEY`
-and `MYUSA_SECRET` keys listed in `.env` are up to date.
-For more on environmental variables and keys, refer to [Required Keys](#required-keys) above.
+  For Homepage url:
 
-If you are not part of 18F and would like to run the application locally, you can
-follow these steps:
-
-1. Create a [MyUSA Account](https://alpha.my.usa.gov/) and create an application for
-development with the following:
+  `http://localhost:5000/`
 
-  For Url:
+  Authorization callback URL:
 
-  `http://localhost:5000/`
+  `http://localhost:5000/auth/githubteammember/callback`
 
-  For Redirect uri:
+1. Save the client id and client secret from your new application in the `.env`
+file in the app directory as `GITHUB_CLIENT_ID` and `GITHUB_CLIENT_SECRET` (see
+.sample.env for formatting help)
 
-  `http://localhost:5000/auth/myusa/callback`
+1. Edit the `GITHUB_TEAM_ID` in your local `.env` file to match the id of a
+GitHub team that you belong to.
 
-2. Under `Select the API Scopes that your Application will use`: select `Email
-   Address`.
+The easiest way to find a Team ID on GitHub is to generate an [access
+token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/)
+and then curl with it:
 
-3. Generate a set of keys by clicking `New Api Key` next to your application in MyUSA.
-   They will be called `Consumer Public Key` and `Consumer Secret Key` on MyUSA but will
-   map to `MYUSA_KEY` and `MYUSA_SECRET`, in your local `.env` file.
+```bash
+curl https://api.github.com/user/teams?access_token=YOUR_ACCESS_TOKEN
+```
 
-4. Edit the `AUTH_DOMAIN` value in your local `.env` file such that the `is_permitted` method in
-   `/app/controllers/auth_controller` will accept the email address you used in your MyUSA
-   application.
-  For example, if you use gmail, change the AUTH_DOMAIN to "gmail.com"
+If you do not belong to a GitHub team or would like to set up the app without
+doing the above, an alternative is to comment out the line in the controller
+that confirms you are a member of the correct team:
 
 ```ruby
-  # Invocation
-  if is_permitted?(auth_email)
-
-  # Method
-  def is_permitted?(auth_email)
-    /#{ENV['AUTH_DOMAIN']}/.match(auth_email)
+  # app/controllers/auth_controller
+
+  def oauth_callback
+    # if team_member?
+      dev_test_email = your_email@example.com
```

Comment:
I realise that most people are going to know the right thing to do here, but how about putting that email address in quotes?